src/scala/com/github/johnynek/bazel_deps/Decoders.scala

```diff
@@ -15,6 +15,12 @@ object Decoders {
       case "runtime_deps" => Right(Transitivity.RuntimeDeps)
       case other => Left(s"unrecogized transitivity: $other")
     }
+  implicit val resolverCacheDecoder: Decoder[ResolverCache] =
+    Decoder.decodeString.emap {
+      case "local" => Right(ResolverCache.Local)
+      case "bazel_output_base" => Right(ResolverCache.BazelOutputBase)
```

Comment:
I guess you could have `resolverCache` be `Either[Map[String, String], String]` so we could have:
```
resolverCache: local
# or
resolverCache: bazel_output_path
# or
resolverCache:
  path: "some/relative-path"
# or
resolverCache:
  path: "/some/abs/path"
```
but maybe this makes it easy to confuse the second case for the first. Maybe your suggestion is the best. I'm not crazy about `computed: local` since the "computation" here is really trivial.

Ugh... bikesheds... 

Let's just merge yours as is and if someone cares enough to add `path` support we can either do the sentinel or Either approach at that time. What do you think?